x-pack/plugins/security/server/routes/analytics/record_violations.ts

```diff
@@ -91,7 +91,7 @@ export const permissionsPolicyViolationReportSchema = schema.object(
         /**
          * The string identifying the policy-controlled feature whose policy has been violated. This string can be used for grouping and counting related reports.
          */
-        featureId: schema.string(),
+        policyId: schema.string(),
```

Comment:
Probably @azasypkin could use your advice here, the [spec](https://www.w3.org/TR/permissions-policy/#reporting) mentions `featureId`, however the report that is sent from Chrome (my version is `126.0.6478.127`) has `policyId`, not `featureId`. Should we support both?

Reply:
Added support for both ✅